orchestration/src/main/java/com/sap/ai/sdk/orchestration/ConfigToRequestTransformer.java

```diff
@@ -15,9 +15,9 @@
 import lombok.NoArgsConstructor;
 import lombok.val;
 
-/** Factory to create all data objects from an orchestration configuration. */
+/** Internal factory to create all data objects from an orchestration configuration. */
 @NoArgsConstructor(access = AccessLevel.NONE)
-final class ConfigToRequestTransformer {
+public final class ConfigToRequestTransformer {
```

Comment:
(Major) 

You are publishing this API.
Please add `@Beta`
Please add `@since` in JavaDoc

Reply:
I unpublished it